CHANGELOG.md

```diff
@@ -1,16 +1,17 @@
-- Allow the user to verify the authenticity of a Zowe driver. The script `zowe-verify-authenticity.sh` will check that a Zowe `ROOT_DIR` for an installed release matches the contents for when that release was created, which assists with support and troubleshooting.  A new documentation chapter [Verify Zowe Runtime Directory](https://docs.zowe.org/stable/troubleshoot/verify-fingerprint.html) has been included that describes the operation of the script. To verify pre-1.14 releases, the script and its associated code are available [separately](https://github.com/zowe/zowe-install-packaging/blob/master/files/fingerprint.pax), see [#1552](https://github.com/zowe/zowe-install-packaging/issues/1552).
-- Inclusion of z/OSMF workflows for Zowe z/OS configuration [#1527](https://github.com/zowe/zowe-install-packaging/issues/1527)
-- Warning check if `ZWESVSTC` runs under user ID `IZUSVR` [#1534](https://github.com/zowe/zowe-install-packaging/issues/1534)
+## `1.14.0`
 
+### New features and enhancements
+- Allow the user to verify the authenticity of a Zowe driver. The script `zowe-verify-authenticity.sh` will check that a Zowe `ROOT_DIR` for an installed release matches the contents for when that release was created, which assists with support and troubleshooting. To verify pre-1.14 releases, the script and its associated code are available [separately](https://github.com/zowe/zowe-install-packaging/blob/staging/files/fingerprint.pax) (see [#1552](https://github.com/zowe/zowe-install-packaging/issues/1552)). For more information, see the new topic [Verify Zowe Runtime Directory](https://docs.zowe.org/stable/troubleshoot/verify-fingerprint.html) that describes the operation of the script. 
+- Allow multiple domains (names/IP Addresses) when generating certificates. This also includes SMP/E `HOLDDATA` for the affected function `Zowe Configuration`. [#1511](https://github.com/zowe/zowe-install-packaging/issues/1511)
+- Included z/OSMF workflows for Zowe z/OS configuration. [#1527](https://github.com/zowe/zowe-install-packaging/issues/1527)
+- Added warning if `ZWESVSTC` runs under user ID `IZUSVR`. [#1534](https://github.com/zowe/zowe-install-packaging/issues/1534)
+- [Docs] Changed the documentation so that SZWEAUTH PDSE load library members should not be copied elsewhere, but instead the original installation target SZWEAUTH PDSE should be APF-authorized and used as the runtime load library.  This also includes SMP/E `HOLDDATA` for the affected function `STC JCL` as well as changes to topics [Installing and configuring the Zowe cross memory server (ZWESISTC)](https://docs.zowe.org/stable/user-guide/configure-xmem-server.html) and [Installing and starting the Zowe started task (ZWESVSTC)](https://docs.zowe.org/stable/user-guide/configure-zowe-server.html).  
```

Comment:
It might be clearer to change 
`Changed the documentation so that SZWEAUTH PDSE load library members should not be copied elsewhere` 
to say
`Changed the documentation to say that SZWEAUTH PDSE load library members should not be copied elsewhere`